<launch>


  <node pkg="rosserial_python" type="serial_node.py" name="mega_node">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="115200"/>
      <remap from="/sweeper/l_motor_cmd" to="pwr_cmd_"/>
  </node>
    
    
  <node pkg="sweeper_interface" type = "wheel_PID_controller" name="wheel_PID_controller"  output="screen">
      <remap from="/encoder_ticks" to="/sweeper/lwheel"/>
      <param name="Kp" value="256"/>
      <param name="Ki" value="15"/>
      <param name="Kd" value="50"/>
  </node>

    <node name="PID_Plot"
        pkg="rqt_plot"
        type="rqt_plot"
        args="/wheel_cmd_vel /wheel_vel" />

</launch>
